domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidsforhold/InntektsmeldingTjeneste.java

```diff
@@ -198,14 +201,19 @@ public void lagreInntektsmelding(Saksnummer saksnummer, Long behandlingId, Innte
                                 .anyMatch(ap -> ap.getPeriode().inkluderer(skjæringstidspunktet)
                                         || ap.getPeriode().getTomDato().isAfter(skjæringstidspunktet));
                     });
-            if (skalFjernes && !erAmbasade(im) && !harOppgittFiske(oppgittOpptjening) && arbeidsgiverHarVærtRegistrertIOpplysningsperioden) {
+            if (skalFjernes && !erAmbasade(im) && !harOppgittFiske(oppgittOpptjening) && arbeidsgiverHarVærtRegistrertIOpplysningsperioden && !harOverstyringTilknyttetSeg) {
                 fjernes.add(im);
             }
         });
         kladd.removeAll(fjernes);
         return List.copyOf(kladd);
     }
 
+    private static boolean finnesOverstyringTilknyttetIM(Collection<ArbeidsforholdOverstyring> arbeidsforholdOverstyringer, Inntektsmelding im) {
```

Comment:
Så uansett overstyring så skal IM brukes når det er inaktive arbeisforhold?

Reply:
burde vi i såfall legge inn en kommentar så vi husker hvorfor?

Reply:
Denne filtrerer jo i praksis bare ut manuelt opprettede arbeidsforhold etter at 5080 ble avlivet. Og det er for å hindre at vil filtrerer ut arbeidsforhold som det i dette caset. Siden de setter sluttdato 2 måneder før stp vil arbeidsforhold bli regnet som inaktivt og inntektsmeldingen blir derfor filtrert ut

Reply:
Men ja jeg kan godt legge på en kommentar om hvorfor 👍